app/assets/javascripts/dashboards.js

```diff
@@ -0,0 +1,5 @@
+$(function () {
+  $('#no-deploy-toggler').click(function(e) {
+    $('tr.no-elements').toggleClass('hidden');
```

Comment:
could also do `$("a.toggle").click(function(e){ $(this).data('target').toggle() })` to get something reusable